Integrations/src/test/java/io/deephaven/integrations/learn/IndexSetTest.java

```diff
@@ -0,0 +1,81 @@
+package io.deephaven.integrations.learn;
+
+import junit.framework.TestCase;
+import org.junit.Test;
+
+import java.util.NoSuchElementException;
+import java.util.PrimitiveIterator;
+
+public class IndexSetTest {
+
+    static IndexSet indexSet;
+
+    @Test
+    public void AddVerifyRemoveTest() throws Exception {
+        indexSet = new IndexSet(5);
+
+        TestCase.assertEquals(-1, indexSet.current);
+        TestCase.assertEquals(5, indexSet.maxSize);
```

Comment:
These kinds of asserts come with downsides.  To make them work, you made the fields non-private, which is generally discouraged in java.